docs/src/api/class-locatorassertions.md

```diff
@@ -933,7 +933,7 @@ await Expect(
 * langs:
   - alias-java: containsText
 
-Ensures the [Locator] points to an element that contains the given text. You can use regular expressions for the value as well.
+Ensures the [Locator] points to an element that contains the given text. All children of the element are evaluated for the specified text. You can use regular expressions for the value as well.
```

Comment:
This sounds to me as if Playwright would try each of the child elements and see if it contains the text. How about something like: "All nested elements will be considered when computing the text content of the element." ?

Reply:
Good point; thanks for the suggestion. I will implement the change.